mergify_engine/github_events.py

```diff
@@ -35,6 +36,15 @@
 LOG = daiquiri.getLogger(__name__)
 
 
+async def store_active_users(
+    redis_cache: utils.RedisCache, repo: github_types.GitHubRepository
+) -> None:
+    id_owner = repo["owner"]["id"]
+    id_repo = repo["id"]
+    await redis_cache.hset(id_owner, "timestamp", time.time())
```

Comment:
We have many other things stored un Redis, so we should prefix it with something like `f"active-users~{id_owner}"`

Reply:
We also need to store who has done the action we received as an event. For example in this event: https://docs.github.com/en/developers/webhooks-and-events/events/github-event-types#pullrequestevent we want to track who has open/edit/close/... the pull request and when. (e.g. the `["pull_request"]["author"]["login"]` attribute of the event)